start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
Actually, the interact/manipulate GUI parameters can be further offloaded to Jupyter (ref: https://ipywidgets.readthedocs.io/en/latest/examples/Using%20Interact.html). So as to approach http://distill.pub/2017/momentum/.